src/payment/strategies/googlepay/googlepay-adyenv3-payment-processor.ts

```diff
@@ -0,0 +1,77 @@
+import { some } from 'lodash';
+
+import { Payment, PaymentActionCreator, PaymentInitializeOptions } from '../..';
+import { CheckoutStore, InternalCheckoutSelectors } from '../../../checkout';
+import { MissingDataError, MissingDataErrorType, NotInitializedError, NotInitializedErrorType, RequestError } from '../../../common/error/errors';
+import { PaymentMethodCancelledError } from '../../errors';
+import { AdyenAction, AdyenAdditionalAction, AdyenAdditionalActionState, AdyenClient, AdyenError, AdyenV3ScriptLoader } from '../adyenv3';
+
+export default class GooglePayAdyenV3PaymentProcessor {
```

Comment:
Didn't find a place, where do you use this class. It only used in commented code above. Do you need this one?